sdk/state/payment.go

```diff
@@ -107,6 +107,30 @@ type CloseAgreement struct {
 	Transactions CloseTransactions
 }
 
+func (ca CloseAgreement) SignedTransactions() CloseTransactions {
+	declTx := ca.Transactions.Declaration
```

Comment:
💡  I think we should add a check for if `declTx != nil` and `closeTx != nil` since those are pointers. Before I think it was safe because the line `txs, err := c.closeTxs(c.openAgreement.Envelope.Details, cae.Details)` was included which either gave the declTx and closeTx or would error

Reply:
Good point. That raises the question what should this function do if it contains no transactions, or only some. Should it return an error, or simply return signed transactions for what it has. I'm tempted to say we should do the latter so we don't have yet another function that returns errors 😄.

Reply:
> Should it return an error, or simply return signed transactions for what it has
ya  I think return signed transactions for what it has makes sense. And then the code calling this function can check for nil there

Reply:
After looking at this more I don't think the transactions are the issue. The SDK guarantees that whenever a CloseAgreement is returned from any Propose or Confirm function, it contains all the transactions. While they are pointers, they aren't intended to be optional.

However, on further inspection the signatures are optional. ProposePayment returns a CloseAgreement that only has ProposerSignatures. ConfirmerSignatures are optional. I've added support for getting signed transactions without confirmer signatures so that it is safe to call this function all the time. cc415cc